strawberry/django/views.py

```diff
@@ -23,6 +24,10 @@
 from .context import StrawberryDjangoContext
 
 
+class TemporalHttpResponse(HttpResponse):
```

Comment:
I just realized it's better to inherit from `JsonResponse` here.
As `Content-Type` header will be overwritten  from json to `text/html` in line 99 below

Reply:
good spot! sending a PR now